app/scripts/modules/titus/src/serverGroup/configure/wizard/pages/disruptionBudget/JobDisruptionBudget.tsx

```diff
@@ -165,103 +178,106 @@ export class JobDisruptionBudget extends React.Component<IJobDisruptionBudgetPro
                 text={<b>Use Netflix Defaults</b>}
               />
             )}
+            layout={runJobView ? ({ input }) => <>{input}</> : undefined}
           />
 
-          <div style={{ opacity: usingDefault ? 0.5 : 1 }}>
-            <div className="sp-formGroup">
-              <div className="groupHeader">
-                <FormikFormField
-                  name="policyType"
-                  label="Policy"
-                  input={(props) => (
-                    <div>
-                      <TetheredSelect
-                        {...props}
-                        menuContainerStyle={{ height: '300px' }}
-                        className="Select-menu-long"
-                        style={{ width: '300px' }}
-                        disabled={usingDefault}
-                        clearable={false}
-                        onChange={this.policyTypeChanged}
-                        value={policyType}
-                        options={policyOptions}
-                      />
-                      <HelpField
-                        expand={true}
-                        content="A job policy defines container relocation rules and constraints"
-                      />
-                    </div>
-                  )}
-                />
-              </div>
-              <div className="sp-formItem">
-                <p>{policyType.description}</p>
-              </div>
-              {PolicyFields && <PolicyFields isDisabled={usingDefault} />}
-            </div>
-            {!isSelfManaged && (
-              <div>
-                <div className={`${budget.rateUnlimited ? '' : 'sp-formGroup'}`}>
-                  <div className={`${budget.rateUnlimited ? '' : 'groupHeader'}`}>
-                    <FormikFormField
-                      name="rates"
-                      label="Rates"
-                      input={(props) => (
+          {showConfig && (
```

Comment:
I only added this boolean check.

Reply:
Why?

Reply:
I was just pointing that out because the PR makes it seem like I added everything after line 184

Reply:
But I added the boolean because it says whether or not to automatically show the extra config or not. (The gif shows an example of that). In the deploy stage, this is always visible but in the run job stage it can be toggled.